<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtArteryTechNonlinearSolverSetupUI</class>
 <widget class="QDialog" name="QtArteryTechNonlinearSolverSetupUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>243</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtArteryTechNonlinearSolverSetupUI</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_9">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>非线性求解器类型</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="NonlinearSolverTypeLabel">
             <property name="text">
              <string>Nonlinear Solver Type:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLineEdit" name="NonlinearSolverTypeLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>非线性收敛相对误差</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="RelativeTolleranceLabel">
             <property name="text">
              <string>Relative Tollerance:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLineEdit" name="RelativeTolleranceLineEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>非线性收敛绝对误差</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="AbsoluteTolleranceLabel">
             <property name="text">
              <string>Absolute Tollerance:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLineEdit" name="AbsoluteTolleranceLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>最大非线性迭代次数</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="MaxIterationNumberLabel">
             <property name="text">
              <string>Max Iteration Number:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLineEdit" name="MaxIterationNumberLineEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>监视非线性求解器</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="MonitorNonlinearSolverLabel">
             <property name="text">
              <string>Monitor Nonlinear Solver:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLineEdit" name="MonitorNonlinearSolverLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>重建雅可比行列式</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="RebuildJacobianLabel">
             <property name="text">
              <string>Rebuild Jacobian:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLineEdit" name="RebuildJacobianLineEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>重建预处理器</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="RebuildPreconditionerLabel">
           <property name="text">
            <string>Rebuild Preconditioner:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLineEdit" name="RebuildPreconditionerLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QPushButton" name="OkPushButton">
         <property name="text">
          <string>确定</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="CancelPushButton">
         <property name="text">
          <string>取消</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
